memaccess/memaccess.go

```diff
@@ -19,35 +19,35 @@ func (m MemoryRegion) String() string {
 	return fmt.Sprintf("MemoryRegion[%x-%x)", m.Address, m.Address+uintptr(m.Size))
 }
 
-// A centinel value indicating that there is no more regions available.
+// NoRegionAvailable is a centinel value indicating that there is no more regions available.
 var NoRegionAvailable MemoryRegion
 
 // NextReadableMemoryRegion returns a memory region containing address, or the next readable region after address in
```

Comment:
It is resolve following golint warning.

> warning: comment on exported var NoRegionAvailable should be of the form "NoRegionAvailable ..." (golint)

Any comment for exported variable or function should start with name of identifier.